core/pkg/eval/json_evaluator.go

```diff
@@ -321,7 +323,8 @@ func (je *JSONEvaluator) evaluateVariant(reqID string, flagKey string, context m
 		}
 
 		context = je.setFlagdProperties(context, flagdProperties{
-			FlagKey: flagKey,
+			FlagKey:   flagKey,
+			Timestamp: time.Now(),
```

Comment:
I think `time.Now()` returns a `Time` type, which is useful in Go, but not very useful in JSONLogic rules.

Consistent with flagd's unix-ish, and bare-bones philosophy, I think we should use `time.Now().Unix()` which will return a unix timestamp. I don't think any more precision than seconds since 1970 is needed - I cant imagine anyone would write a rule that needs to be precise to the nanosecond.

This will allow users to write rules which conceptually say `if $flagd.timestamp > {my-release-unix-timestamp} return {my-variant}`.

cc @beeme1mr @thisthat @Kavindu-Dodan @bacherfl

Reply:
Excellent point.  Updated to use a unix timestamp.

Reply:
+1 for the timestamp

Reply:
Also +1 on my side for unix timestamp 👍